manager_dashboard/manager_dashboard/js/forms.js

```diff
@@ -30,6 +30,23 @@ function displayProjectTypeFormular(projectType) {
   }
 }
 
+function addTileServerCredits (tileServer, projectType, which) {
+    if (tileServer == 'bing') {
+        document.getElementById("tileServerCredits"+projectType+which).value = "© 2019 Microsoft Corporation, Earthstar Geographics SIO"
+    } else if (tileServer == 'maxar_premium' | tileServer == 'maxar_standard') {
+        document.getElementById("tileServerCredits"+projectType+which).value = "© 2019 Maxar"
+    } else if (tileServer == 'esri' | tileServer == 'esri_world') {
+        document.getElementById("tileServerCredits"+projectType+which).value = "© 2019 ESRI"
+    } else if (tileServer == 'mapbox') {
+        document.getElementById("tileServerCredits"+projectType+which).value = "© 2019 MapBox"
+    } else if (tileServer == 'sinergise') {
+        document.getElementById("tileServerCredits"+projectType+which).value = "© 2019 Sinergise"
+    } else if (tileServer == 'custom') {
+        document.getElementById("tileServerCredits"+projectType+which).value = "Please add imagery credits here."
+    }
```

Comment:
This works, but you can make it easier to read with something like a [switch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) or using a mapping like this:
```javascript
credits = {
"bing": "bing credit text",
"esri": "esri credit text"
// ... the other ones
};
document.getElementById("tileServerCredits"+projectType+which).value = credits[tileServer];
```
It works well here because you only have a handful of fixed values. It tends to be much more maintainable, because you write everything just once.

Reply:
I like your approach. It is much simpler.

Reply:
yes, it does look a lot cleaner now!